flow/barito_consumer_service.go

```diff
@@ -85,9 +90,15 @@ func NewBaritoConsumerService(params map[string]interface{}) BaritoConsumerServi
 		elasticPassword:        params["elasticPassword"].(string),
 	}
 
+	httpClient := http.DefaultClient
+	// if using mTLS, create new http client with tls config
+	if _, ok := params["elasticCaCrt"]; ok {
+		httpClient = s.newHttpClientWithTLS(params["elasticCaCrt"].(string), params["elasticClientCrt"].(string), params["elasticClientKey"].(string))
+	}
```

Comment:
```suggestion
        httpClient := &http.Client{}
	// if using mTLS, create new http client with tls config
	if _, ok := params["elasticCaCrt"]; ok {
		httpClient = s.newHttpClientWithTLS(params["elasticCaCrt"].(string), params["elasticClientCrt"].(string), params["elasticClientKey"].(string))
	}
```

using `http.DefaultClient` directly is dangerous moreover if you have applied changes to it (adding TLS), since it is [a reference type](https://cs.opensource.google/go/go/+/refs/tags/go1.21.2:src/net/http/client.go;l=110). It might affect other line of code that are using `http.DefaultClient`